lib/screens/cart_page.dart

```diff
@@ -655,6 +659,98 @@ class _CartPageState extends State<CartPage> {
     );
   }
 
+  Widget _buildAboutDeliveryMethodInfo() {
+    return Container(
+      decoration: BoxDecoration(
+        border: Border(top: BorderSide(color: Color(0xFFA2A2A2))),
+      ),
+      child: Padding(
+        padding: EdgeInsets.symmetric(vertical: 37),
+        child: Column(
+          children: <Widget>[_buildAboutDeliveryMethod()],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildAboutDeliveryMethod() {
+    return Column(
+      children: <Widget>[
+        Container(
+          margin: const EdgeInsets.only(top: 17.0, bottom: 57.0),
+          child: Text(
+            '配送方法について',
+            style: TextStyle(
+              fontSize: 18,
+              color: paletteBlackColor,
+              fontWeight: FontWeight.w600,
+            ),
+          ),
+        ),
+        Container(
+          child: Text(
+              'ゆうパックもしくは宅急便にて配送致します。\n一律500円の送料がかかります。5,000円以上のお買い上げで送料が無料になります。）',
+              textAlign: TextAlign.justify,
+              style: TextStyle(
+                  color: paletteBlackColor,
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.w300)),
+        ),
+        Container(
+          margin: const EdgeInsets.only(top: 37.0),
+          child: RichText(
+              textAlign: TextAlign.justify,
+              text: TextSpan(
+                  text: '個人情報保護方針',
+                  style: TextStyle(
+                      color: paletteDarkRedColor,
+                      fontSize: 15.0,
+                      fontWeight: FontWeight.w300),
```

Comment:
Instead of setting the text directly in the TextSpan:

```
text: '個人情報保護方針',
                  style: TextStyle(
                      color: paletteDarkRedColor,
                      fontSize: 15.0,
                      fontWeight: FontWeight.w300),
```
why not set it in the children for uniformity? Also, since you are using TextSpan, you can let the children of the TextSpan inherit the root style, so you don't have to define the **color**, **fontSize** and **fontWeight** parameters again and again, since they are same for all the children. The color changes for some children, so you can override the **color** property just for those children. Here is a sample implementation in which I set the default color to **paletteBlackColor** and overridden it in some children to **paletteDarkRedColor**:
```
RichText(
            textAlign: TextAlign.justify,
            text: TextSpan(
              style: TextStyle(
                color: paletteBlackColor,
                fontSize: 15.0,
                fontWeight: FontWeight.w300,
              ),
              children: <TextSpan>[
                TextSpan(
                  text: '個人情報保護方針',
                  style: TextStyle(
                    color: paletteDarkRedColor,
                  ),
                  recognizer: TapGestureRecognizer()
                    ..onTap = () {
                      Navigator.push(
                        context,
                        MaterialPageRoute(
                          builder: (context) => DeliveryPrivacyPolicy(),
                        ),
                      );
                    },
                ),
                TextSpan(
                  text: 'と',
                ),
                TextSpan(
                  text: '利用規約',
                  style: TextStyle(
                    color: paletteDarkRedColor,
                  ),
                  recognizer: TapGestureRecognizer()
                    ..onTap = () {
                      Navigator.push(
                        context,
                        MaterialPageRoute(
                          builder: (context) => DeliveryTermsOfService(),
                        ),
                      );
                    },
                ),
                TextSpan(
                  text: 'に同意して注文',
                ),
              ],
            ),
          )
```